Build instructions fail

This is with go1.10.3 under Ubuntu 16.04

The build instructions in README.md must be missing a step:

```
root@jocko:~# go get github.com/travisjeffery/jocko
package github.com/travisjeffery/jocko: no Go files in /root/go/src/github.com/travisjeffery/jocko
root@jocko:~# ls /root/go/src/github.com/travisjeffery/jocko/
CONTRIBUTING.md  Gopkg.lock  LICENSE   README.md  cmd        docker-compose.yml  log   protocol
Dockerfile       Gopkg.toml  Makefile  _examples  commitlog  jocko               mock  testutil
root@jocko:~# cd ~/go/src/github.com/travisjeffery/jocko
root@jocko:~/go/src/github.com/travisjeffery/jocko# make
commitlog/compact_cleaner.go:4:2: cannot find package "github.com/cespare/xxhash" in any of:
	/usr/local/go/src/github.com/cespare/xxhash (from $GOROOT)
	/root/go/src/github.com/cespare/xxhash (from $GOPATH)
jocko/broker.go:17:2: cannot find package "github.com/davecgh/go-spew/spew" in any of:
	/usr/local/go/src/github.com/davecgh/go-spew/spew (from $GOROOT)
	/root/go/src/github.com/davecgh/go-spew/spew (from $GOPATH)
jocko/metrics.go:3:8: cannot find package "github.com/go-kit/kit/metrics/prometheus" in any of:
	/usr/local/go/src/github.com/go-kit/kit/metrics/prometheus (from $GOROOT)
	/root/go/src/github.com/go-kit/kit/metrics/prometheus (from $GOPATH)
jocko/testing.go:11:2: cannot find package "github.com/hashicorp/consul/testutil/retry" in any of:
	/usr/local/go/src/github.com/hashicorp/consul/testutil/retry (from $GOROOT)
	/root/go/src/github.com/hashicorp/consul/testutil/retry (from $GOPATH)
jocko/fsm/fsm.go:10:2: cannot find package "github.com/hashicorp/go-memdb" in any of:
	/usr/local/go/src/github.com/hashicorp/go-memdb (from $GOROOT)
	/root/go/src/github.com/hashicorp/go-memdb (from $GOPATH)
jocko/broker.go:18:2: cannot find package "github.com/hashicorp/raft" in any of:
	/usr/local/go/src/github.com/hashicorp/raft (from $GOROOT)
	/root/go/src/github.com/hashicorp/raft (from $GOPATH)
jocko/broker.go:19:2: cannot find package "github.com/hashicorp/raft-boltdb" in any of:
	/usr/local/go/src/github.com/hashicorp/raft-boltdb (from $GOROOT)
	/root/go/src/github.com/hashicorp/raft-boltdb (from $GOPATH)
jocko/broker.go:20:2: cannot find package "github.com/hashicorp/serf/serf" in any of:
	/usr/local/go/src/github.com/hashicorp/serf/serf (from $GOROOT)
	/root/go/src/github.com/hashicorp/serf/serf (from $GOPATH)
jocko/testing.go:13:2: cannot find package "github.com/mitchellh/go-testing-interface" in any of:
	/usr/local/go/src/github.com/mitchellh/go-testing-interface (from $GOROOT)
	/root/go/src/github.com/mitchellh/go-testing-interface (from $GOPATH)
jocko/broker.go:21:2: cannot find package "github.com/opentracing/opentracing-go" in any of:
	/usr/local/go/src/github.com/opentracing/opentracing-go (from $GOROOT)
	/root/go/src/github.com/opentracing/opentracing-go (from $GOPATH)
jocko/broker.go:22:2: cannot find package "github.com/pkg/errors" in any of:
	/usr/local/go/src/github.com/pkg/errors (from $GOROOT)
	/root/go/src/github.com/pkg/errors (from $GOPATH)
jocko/broker.go:23:2: cannot find package "github.com/satori/go.uuid" in any of:
	/usr/local/go/src/github.com/satori/go.uuid (from $GOROOT)
	/root/go/src/github.com/satori/go.uuid (from $GOPATH)
cmd/jocko/main.go:9:2: cannot find package "github.com/spf13/cobra" in any of:
	/usr/local/go/src/github.com/spf13/cobra (from $GOROOT)
	/root/go/src/github.com/spf13/cobra (from $GOPATH)
cmd/jocko/main.go:10:2: cannot find package "github.com/tj/go-gracefully" in any of:
	/usr/local/go/src/github.com/tj/go-gracefully (from $GOROOT)
	/root/go/src/github.com/tj/go-gracefully (from $GOPATH)
jocko/testing.go:14:2: cannot find package "github.com/travisjeffery/go-dynaport" in any of:
	/usr/local/go/src/github.com/travisjeffery/go-dynaport (from $GOROOT)
	/root/go/src/github.com/travisjeffery/go-dynaport (from $GOPATH)
commitlog/index.go:12:2: cannot find package "github.com/tysontate/gommap" in any of:
	/usr/local/go/src/github.com/tysontate/gommap (from $GOROOT)
	/root/go/src/github.com/tysontate/gommap (from $GOPATH)
jocko/testing.go:20:2: cannot find package "github.com/uber/jaeger-client-go" in any of:
	/usr/local/go/src/github.com/uber/jaeger-client-go (from $GOROOT)
	/root/go/src/github.com/uber/jaeger-client-go (from $GOPATH)
jocko/testing.go:21:2: cannot find package "github.com/uber/jaeger-client-go/config" in any of:
	/usr/local/go/src/github.com/uber/jaeger-client-go/config (from $GOROOT)
	/root/go/src/github.com/uber/jaeger-client-go/config (from $GOPATH)
cmd/jocko/main.go:19:2: cannot find package "github.com/uber/jaeger-client-go/log" in any of:
	/usr/local/go/src/github.com/uber/jaeger-client-go/log (from $GOROOT)
	/root/go/src/github.com/uber/jaeger-client-go/log (from $GOPATH)
jocko/testing.go:18:2: cannot find package "github.com/uber/jaeger-lib/metrics" in any of:
	/usr/local/go/src/github.com/uber/jaeger-lib/metrics (from $GOROOT)
	/root/go/src/github.com/uber/jaeger-lib/metrics (from $GOPATH)
jocko/structs/structs.go:6:2: cannot find package "github.com/ugorji/go/codec" in any of:
	/usr/local/go/src/github.com/ugorji/go/codec (from $GOROOT)
	/root/go/src/github.com/ugorji/go/codec (from $GOPATH)
log/logger.go:6:2: cannot find package "go.uber.org/zap" in any of:
	/usr/local/go/src/go.uber.org/zap (from $GOROOT)
	/root/go/src/go.uber.org/zap (from $GOPATH)
log/logger.go:7:2: cannot find package "go.uber.org/zap/zapcore" in any of:
	/usr/local/go/src/go.uber.org/zap/zapcore (from $GOROOT)
	/root/go/src/go.uber.org/zap/zapcore (from $GOPATH)
Makefile:30: recipe for target 'test' failed
make: *** [test] Error 1
```

Possibly related to #109

--- a/README.md
+++ b/README.md
@@ -69,8 +69,11 @@ Kafka/distributed commit log service in Go.
 
     ```
     $ cd $GOPATH/src/github.com/travisjeffery/jocko
-    $ make
+    $ make build
     ```
+
+    (If you see an error about `dep` not being found, ensure that
+    `$GOPATH/bin` is in your `PATH`)
 
 ### Docker
 

